app/controllers/offers_controller.rb

```diff
@@ -22,8 +22,9 @@ def index
     end
 
     @offers_with_coordinates = @offers.where.not(latitude: nil).and(@offers.where.not(longitude: nil))
+    @offers_in_europe = @offers_with_coordinates.near("Berlin", 20000, min_radius: 1)
```

Comment:
Will this still care of situations where latitude or longitude is nil?

Reply:
I think it should be fine, it is only used when opening the /offers view for the first time. So that not the whole world map is shown. Then again, now that I am thinking - 20.000 is way too much :D 

Usually the offers with nil latitude and longitude are placed next to Australia or so. When I tested it, this tool care of the issue but let's see with the current offers from the API. 

I noted it as a "to check" in the Trello board.